prow/cmd/mkpj/main.go

```diff
@@ -204,6 +211,9 @@ func gatherOptions() options {
 	fs.IntVar(&o.pullNumber, "pull-number", 0, "Git pull number under test")
 	fs.StringVar(&o.pullSha, "pull-sha", "", "Git pull SHA under test")
 	fs.StringVar(&o.pullAuthor, "pull-author", "", "Git pull author under test")
+	fs.BoolVar(&o.triggerJob, "trigger-job", false, "Submit the job to Prow and wait for results")
+	fs.StringVar(&o.outputPath, "output-path", "", "File to which output of the triggered job would be printed")
```

Comment:
What for do we need that? Doesn't it make more sense to simply print the job on stdout and print the progress stuff on stderr? Thats how other popular tools like e.G. `curl` work.

If really wanted, ppl can redirect the `stderr` stream to a file

Reply:
If no file is provided, I am printing it by default onto stdout, should I remove this option altogether?

Reply:
yeah please, that will simplify the code and as mentioned, if ppl really want this, they can simply redirect the stream

Reply:
alright i'll do that